edit-post/api/test/more-menu-item.js

```diff
@@ -0,0 +1,29 @@
+import '@wordpress/jest-console';
+
+let registerMoreMenuItem, getMoreMenuItems, registerSidebar;
+function requireAll() {
+	jest.resetModules();
+	const sidebar = require( '../sidebar' );
+	registerSidebar = sidebar.registerSidebar;
+	const moreMenuItem = require( '../more-menu-item' );
+	registerMoreMenuItem = moreMenuItem.registerMoreMenuItem;
+	getMoreMenuItems = moreMenuItem.getMoreMenuItems;
+}
+
+requireAll();
+
+describe( 'registerMoreMenuItem', () => {
+	beforeEach( requireAll );
```

Comment:
Every test suite is executed in isolation. There is only one test in this file, so it is totally fine to import all dependencies using `import` statements in the same way as you would do it in production code.

Reply:
I've added another test, because I do believe that if we would add more tests it's important that they are executed in a clean context.

Reply:
Worth pointing out that resetting the Node `require` cache on every test case is not scalable, and its use should be limited.